test/unit/autoNumeric.spec.js

```diff
@@ -1379,15 +1379,30 @@ describe('autoNumeric options and `options.*` methods', () => {
             expect(aNInput.getFormatted()).toEqual('');
             expect(aNInput.getNumber()).toEqual(0);
 
-            aNInput.set('');
+            aNInput.options.reset().french().set('');
             aNInput.options.emptyInputBehavior(AutoNumeric.options.emptyInputBehavior.press);
             expect(aNInput.getFormatted()).toEqual('');
+            aNInput.options.reset().french().set('');
             aNInput.options.emptyInputBehavior(AutoNumeric.options.emptyInputBehavior.focus);
             expect(aNInput.getFormatted()).toEqual('');
+            aNInput.options.reset().french().set('');
             aNInput.options.emptyInputBehavior(AutoNumeric.options.emptyInputBehavior.always);
             expect(aNInput.getFormatted()).toEqual('\u202f€');
+            aNInput.options.reset().french().set('');
+            aNInput.options.emptyInputBehavior(AutoNumeric.options.emptyInputBehavior.min);
+            expect(aNInput.getFormatted()).toEqual('-9.999.999.999.999,99\u202f€');
```

Comment:
This is great :)

Could you please add a test for each `rawValue` starting from here in that test block?

For example for the line above:
```js
expect(aNInput.getNumericString()).toEqual('-9999999999999.99');
```